### PR TITLE
Support getmempoolinfo.

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

**Comment by davecgh** [comment]
>	rpcserverhelp_test.go:17: RPC handler defined for method 'getmempoolinfo' without also specifying result types
>	rpcserverhelp_test.go:44: Failed to generate help for method 'getmempoolinfo': no result types specified for method getmempoolinfo

File: rpcserver.go [hunk 1]
```diff
@@ -1938,6 +1939,24 @@ func handleGetInfo(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (in
 	return ret, nil
 }
 
+// handleGetMempoolInfo implements the getmempoolinfo command.
+func handleGetMempoolInfo(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	mp := s.server.txMemPool
+	txD := mp.TxDescs()
```
Comment (davecgh) [inline rpcserver.go]: I'd combine these two lines.  The `mp` local isn't used anywhere else.

**Comment by davecgh** [comment]
As Travis pointed out, the help text for the command is missing.

**Comment by dajohi** [comment]
OK

**Comment by davecgh** [comment]
I still need to test it out, but it reads well now.  Thanks!

**Comment by davecgh** [comment]
Also, the [JSON-RPC API docs](https://github.com/btcsuite/btcd/blob/master/docs/json_rpc_api.md) need to be updated to include the new command.

**Comment by dajohi** [comment]
Updated.

**Comment by davecgh** [comment]
Updates look good.  Thanks.

**Comment by davecgh** [comment]
Tested and confirmed it's working properly.  OK.

**Pull request merged**